cmd/zetaclientd/debug.go

```diff
@@ -57,7 +56,7 @@ func debugCmd(_ *cobra.Command, args []string) error {
 		return err
 	}
 
-	appContext := clientcontext.New(cfg, zerolog.Nop())
+	appContext := clientcontext.NewAppContext(cfg)
```

Comment:
We should rather continue to use `New`
I think this change was introduced by @swift1337 ?

Reply:
https://github.com/zeta-chain/node/pull/2411/commits/4f61c380bd8629ea044d7284eabf99c09b447131